Pearson-Advance/tutor-ecommerce
# PADV-523 - Making ecommerce MFEs optional.

## Description:

This PR adds the possibility to disable the ecommerce MFEs.

## Test:

- Add these overwrite to a Tutor plugin: `"ECOMMERCE_PAYMENT_MFE_APP": None, "ECOMMERCE_MFE_APP": None,`
- Run tutor config save.
- Check that the MFEs Dockerfile is rendered without these MFEs.

## Reviewers:
